tests/tests_integration/test_api/test_datapoints.py

```diff
@@ -421,10 +421,10 @@ def test_retrieve_outside_points__query_limit_plusminus1_tests(
         "start, end, exp_first_ts, exp_last_ts",
         # fmt: off
         [
-            (631670400000 + 1, 693964800000,     631670400000,           693964800000),
-            (631670400000,     693964800000,     631670400000 - WEEK_MS, 693964800000),
-            (631670400000,     693964800000 + 1, 631670400000 - WEEK_MS, 693964800000 + WEEK_MS),
-            (631670400000 + 1, 693964800000 + 1, 631670400000,           693964800000 + WEEK_MS),
+            (631670400000 + 1, 693964800000, 631670400000, 693964800000),
```

Comment:
Let’s also remove the `fmt on/off `comments since it doesn’t seem to respect them.

Reply:
It does respect it actually. It was just an artefact of running black without proper config.

Reply:
Fixed!

Reply:
Hmm. It's supposed to https://docs.astral.sh/ruff/formatter/#format-suppression.

Reply:
Removed though.